packages/happy-css-modules/src/transformer/scss-transformer.ts

```diff
@@ -22,7 +22,8 @@ function promisifySassRender(sass: typeof import('sass')) {
 export const createScssTransformer: () => Transformer = () => {
   let sass: typeof import('sass');
   return async (source, options) => {
-    sass ??= (await import('sass').catch(handleImportError('sass'))).default;
+    const importedSass = await import('sass').catch(handleImportError('sass'))
+    sass ??= importedSass.default ?? importedSass;
```

Comment:
note: I implemented this pattern because I confirmed that simply removing .default would cause it to stop working with versions prior to 1.60.x.